src/clarity_vm/database/marf.rs

```diff
@@ -47,11 +47,14 @@ impl MarfedKV {
             .ok_or_else(|| InterpreterError::BadFileName)?
             .to_string();
 
+        let mut marf_opts = MARFOpenOpts::default();
```

Comment:
Should this take the `MARFOpenOpts` as a parameter, and default to `MARFOpenOpts::default() + external_blobs = true`?

Reply:
Sure, I can plumb this through.  I'll have to make it so that we can extract the open options from the `stacks-node::config` file though, so it'll add about 100-200 extra lines to the diff.